### ISSUE TITLE
Indiquer le nom des partenaires

### ISSUE THREAD

**Issue by cgaspoz** [HEG-Arc/PrestaSoc#61]
@jotabin Je pense également qu'il faut indiquer qu'il s'agit d'une projet de la HES·SO, en particulier de la Haute école de gestion ARC et de la Haute école de travail social et de la santé Lausanne..

**Comment by urosselet** [comment]
@jotabin : J'ai mis la page "A propos" à jour avec une très courte description du projet et l'adresse de contact mail. Voir dans la version dev du site : https://dev.jestime.ch/apropos 
est-ce qu'on met aussi des liens vers les sites de nos écoles respectives ? 
63f7467b40778c230fdeb1ab00bd88bd2927d1ea

**Comment by jotabin** [comment]
oui, merci !!! Pour nous: https://www.eesp.ch Mais rien ne s'affiche chez moi (safari, chrome, etc…) sous à propos, sinon les dessins…
–
Jean-Pierre Tabin, Professeur
Laboratoire de recherche santé - social (LaReSS)
Haute école de travail social et de la santé | EESP | HES·SO
Ch. des Abeilles 14
1010 Lausanne
Tél. : +41 21 651 62 25/00


Derniers articles parus:
– (2017). Quand l’État social profite des immigrés. Services publics, Journal du syndicat suisse des services publics<http://ssp-vpod.ch/site/assets/files/0/08/798/sepu_6_2017.pdf> (98/6), 6-7.
– avec Isabelle Probst, Monika Piecek-Riondel & Céline Perrin (2016). L’invalidité : une position dominée. Revue française des affaires sociales<https://www.cairn.info/revue-francaise-des-affaires-sociales-2016-4.htm>, 4, 89-108.
– avec Frédérique Leresche (2016). Non-recours et travail social. Actualité sociale, 65, 18-19.



Le 17 avr. 2017 à 09:54, urosselet <notifications@github.com<mailto:notifications@github.com>> a écrit :


@jotabin<https://github.com/jotabin> : J'ai mis la page "A propos" à jour avec une très courte description du projet et l'adresse de contact mail. Voir dans la version dev du site : https://dev.jestime.ch/apropos
est-ce qu'on met aussi des liens vers les sites de nos écoles respectives ?
63f7467<https://github.com/HEG-Arc/PrestaSoc/commit/63f7467b40778c230fdeb1ab00bd88bd2927d1ea>

**Comment by urosselet** [comment]
J'ai mis les liens vers les sites.
Est-ce que tu es bien sur la version "développement" du site? Précédée par
"dev" dans l'url:
https://dev.jestime.ch/apropos
Les changements sont d'abords visibles sur cette version, ensuite on les
pousse sur la version publique.
bonne soirée!

**Comment by jotabin** [comment]
ok!
–
Jean-Pierre Tabin, Professeur
Laboratoire de recherche santé - social (LaReSS)
Haute école de travail social et de la santé | EESP | HES·SO
Ch. des Abeilles 14
1010 Lausanne
Tél. : +41 21 651 62 25/00


Derniers articles parus:
– (2017). Quand l’État social profite des immigrés. Services publics, Journal du syndicat suisse des services publics<http://ssp-vpod.ch/site/assets/files/0/08/798/sepu_6_2017.pdf> (98/6), 6-7.
– avec Isabelle Probst, Monika Piecek-Riondel & Céline Perrin (2016). L’invalidité : une position dominée. Revue française des affaires sociales<https://www.cairn.info/revue-francaise-des-affaires-sociales-2016-4.htm>, 4, 89-108.
– avec Frédérique Leresche (2016). Non-recours et travail social. Actualité sociale, 65, 18-19.



Le 17 avr. 2017 à 21:40, urosselet <notifications@github.com<mailto:notifications@github.com>> a écrit :

J'ai mis les liens vers les sites.
Est-ce que tu es bien sur la version "développement" du site? Précédée par
"dev" dans l'url:
https://dev.jestime.ch/apropos
Les changements sont d'abords visibles sur cette version, ensuite on les
pousse sur la version publique.
bonne soirée!
